README.md

```diff
@@ -54,11 +62,14 @@ The Docker volume plugin requires the docker engine to be installed as a prerequ
 Ubuntu users to configure the docker repository and pull the `docker-engine` package from there.
 Ubuntu users can find instructions [here](https://docs.docker.com/engine/installation/linux/ubuntulinux/).
 
-[Docker recommends that the docker engine should start after the plugins.] (https://docs.docker.com/engine/extend/plugin_api/)
+[Docker recommends that the docker engine should start after the plugins.](https://docs.docker.com/engine/extend/plugin_api/)
+
+Starting 0.13 release, no more DEB/RPMs as vDVS(vSphere docker volume service) is available on [Docker store](https://store.docker.com/plugins/e15dc9d5-e20e-4fb8-8876-9615e6e6e852?tab=description)! It requires to have docker 1.13/17.03 and above. 
```

Comment:
Please keep RPM/Deb in this release. You can add a note that DEB/RPMs are deprecated going forward and will not be available

For Docker 1.13/17.x, install vDVS managed plugin as follows;
`docker plugin install --grant-all-permissions --alias vsphere store/vmware/docker-volume-vsphere:<release>` 

Remove "no question asked and pretend the plugin name is 'vsphere' "